lib/JsonpExportMainTemplatePlugin.js

```diff
@@ -4,25 +4,34 @@
 */
 var ConcatSource = require("webpack-sources").ConcatSource;
 
-function JsonpExportMainTemplatePlugin(name) {
-	this.name = name;
+function JsonpExportMainTemplatePlugin(names) {
+	if (names && names.constructor !== Array) {
+		names = [names];
+	}
+	this.names = names.length > 0 ? names : undefined;
 }
 module.exports = JsonpExportMainTemplatePlugin;
 JsonpExportMainTemplatePlugin.prototype.apply = function(compilation) {
 	var mainTemplate = compilation.mainTemplate;
 	compilation.templatesPlugin("render-with-entry", function(source, chunk, hash) {
-		var name = mainTemplate.applyPluginsWaterfall("asset-path", this.name || "", {
-			hash: hash,
-			chunk: chunk
-		});
-		return new ConcatSource(name + "(", source, ");");
+		return new ConcatSource(this.names.map(function(nameSrc) {
+			var name = mainTemplate.applyPluginsWaterfall("asset-path", nameSrc || "", {
+				hash: hash,
+				chunk: chunk
+			});
+			return name + " && " + name + "(", source, ");";
```

Comment:
incorrect. and: `source` should not be executed multiple times. maybe IIFE + temp for multipe exports.